Jenkinsfile

```diff
@@ -139,6 +139,7 @@ melt.trynode('silver-ableC') {
       "ableC-cilk",
       "ableC-tutorials", "ableC-sample-projects",
       "carbles-ai",
+      "ableC-parallel"
```

Comment:
Just for consistency, maybe move this line to after `"ableC-cilk"`, since the last couple lines are things that aren't really extensions.